wiki.py

```diff
@@ -553,3 +554,42 @@ def _add_year_categories(self):
                 else:
                     categories.append(extra_category)
             self._add_category_page(title, categories)
+
+    def _update_current_projects_template(self):
+        """Update the current projects template with the new projects.
+
+        All category pages are added to a category for the year. Some
+        are added to additional categories.
+
+        """
+
+        template_name = self._config["year_pages"]["current_projects_template"]
+        template_content = (
+            "Aktuella projekt/layout|år={year}\n"
+            "|access={Tillgång}\n"
+            "|use={Användning}\n"
+            "|community={Gemenskapen}\n"
+            "|enabling={Möjliggörande}\n"
+        )
+        project_format = "[[{ns}:{{proj}}|{{proj}}]]".format(
+            ns=self._config["project_namespace"])
+        delimiter = "''' · '''"
+
+        page = Page(self._site, template_name)
+        template_data = {}
+        for program in self._programs:
+            projects = set()
+            for strategy in program.get('strategies'):
```

Comment:
This sorts by project number. To get them by name, which I assume is what we want, do something like:
```
            for strategy in program.get('strategies'):
                projects.update([self._projects[number] for number in strategy.get("projects")])
            template_data[program.get('name')] = delimiter.join(
                [project_format.format(proj=project)
                 for project in sorted(projects)])
```

Reply:
They should actually be sorted by project number since these groups them correctly and is how wew present them in other places

Reply:
OK then. I thought it was odd since the numbers weren't present. A reader would probably expect alphabetic sorting.